src/chopper.cpp

```diff
@@ -19,8 +19,11 @@
 #include <chopper/layout/execute.hpp>
 #include <chopper/set_up_parser.hpp>
 #include <chopper/sketch/check_filenames.hpp>
+#include <chopper/sketch/output.hpp>
 #include <chopper/sketch/read_data_file.hpp>
 
+#include <hibf/sketch/compute_sketches.hpp>
+
```

Comment:
I think putting everything into main would cause trouble for raptor?
I can't link against your main, so I need to duplicate everything that happens in the main: https://github.com/seqan/raptor/blob/main/src/layout/raptor_layout.cpp

Ideally, the main should only dispatch, I think (e.g., https://github.com/seqan/raptor/blob/main/src/raptor.cpp).

If, for example, everything happens in `execute`, I can just use and link against `execute`.

Reply:
You are right. I'll check what's best for raptor and add another function thank can be linked against .